app/routes/orders/requested_items.js

```diff
@@ -1,3 +1,5 @@
 import detail from "./detail";
 
-export default detail.extend({});
+export default detail.extend({
+  controllerName: "orders/active_items"
```

Comment:
Can't we delete the `requested_items` page ??

Reply:
There's lots of thing happening in that page.I think it would be better to have separate page page/route for it.

Reply:
what is the `requested_items` items page ? 🤔